_data/simple-icons.json

```diff
@@ -14903,6 +14903,12 @@
             "hex": "DD0B78",
             "source": "https://starship.rs"
         },
+        {
+            "title": "StartMail",
+            "hex": "6573FF",
+            "source": "https://www.startmail.com",
```

Comment:
```suggestion
            "source": "https://www.startmail.com/press",
```
Let's use the media kit from the press page as the source. This will mean updating the icon as well but I think going with the provided logo materials is a better route.

This shows the difference between the current icon in this PR (black) with `StartMail Media Kit/01 StartMail Logo/StartMail Icon/StartMail Icon.svg` in color.
![CleanShot 2024-09-13 at 23 01 24](https://github.com/user-attachments/assets/c6d8218d-48c1-4e63-8cd3-8aff9ae23331)